graphql_api/tests/test_branch.py

```diff
@@ -386,6 +386,7 @@ def test_fetch_path_contents_with_files_and_search_value(
         critical_files.return_value = [CriticalFile("folder/fileB.py")]
 
         data = self.gql_request(query_files, variables=variables)
+        print("dataaa", data)
```

Comment:
interesting

Reply:
🙃 I'd removed this in my local and didn't push